modules/src/wiki.py

```diff
@@ -4,9 +4,10 @@ def process(input, entities):
     output = {}
     try:
         query = entities['wikipedia_search_query'][0]['value']
+        page = wikipedia.page(query)
         data = wikipedia.summary(query, sentences=1)
```

Comment:
We can do it with one call by doing data=page.summary but then it becomes hard to only give the user the first sentence of the summary, so I thought making two calls would be fine. Let me know.